src/styles/main.scss

```diff
@@ -1,3 +1,40 @@
+$white-color: white;
+
 body {
   margin: 0;
 }
+
+.container {
+  display: grid;
+  grid-template-columns: repeat((3), 1fr );
+  justify-items: stretch;
+  div {
+    height: 300px;
+  }
+}
+
+@for $i from 1 through 6 {
+  .container__item:nth-child(#{$i})::after {
+    content: "#{$i}";
+    font-size: 100px;
+    font-family: Arial;
+    color: $white-color;
+    background: rgb((255 - 51*($i - 1)), 0, 0);
+    display: flex;
+    height: 100%;
+    align-items: center;
+    justify-content: center;
```

Comment:
this should not be in :nth-child(#{$i})::after, 
but in .container__item, 
color in :nth-child(#{$i})